app/assets/stylesheets/frontend/frontend_actions.css.scss

```diff
@@ -2280,6 +2290,71 @@ figcaption span.attribution a {
 
 }
 
+.slideshow:hover .slick-arrow {
+
+  opacity: 0.8;
+
+}
+
+.slick-arrow {
+
+  height: auto;
+  width: 50px;
+  height: 100px;
+
+  padding-right: -10px;
+
+  display: block;
+  margin-left: 65px;
+  margin-right: 65px;
+  margin-top: -75px;
+  z-index: 2;
+
+  background-image: url("/dev-assets/arrow.svg") !important;
```

Comment:
```suggestion
  background-image: image-url("arrow.svg") !important;
```

Reply:
This path should not be hard-coded, it's different for development and production (the path in production is /assets/arrow.svg").